sdcm/scylla_bench_thread.py

```diff
@@ -203,6 +207,8 @@ def _run_stress(self, loader, loader_idx, cpu_idx):  # pylint: disable=too-many-
             )
             cmd_runner_name = loader.ip_address
 
+        is_shell_supported = cmd_runner.run(cmd="sh -c true", timeout=self.timeout, retry=0).returncode == 0
```

Comment:
Not being 0 it will raise exception.
Need to use `ignore_status=True` in such a case.

Reply:
> Not being 0 it will raise exception. Need to use `ignore_status=True` in such a case.

fixed.

Reply:
> this is not an assert statement...

?